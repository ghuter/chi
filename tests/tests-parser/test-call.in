x :: f();
y :: f(a);
z :: f(a + 1, b);
