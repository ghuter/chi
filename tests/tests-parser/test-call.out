const SDECL(x, ECALL(EMEM(f)) : ?) : ?
const SDECL(y, ECALL(EMEM(f), EMEM(a)) : ?) : ?
const SDECL(z, ECALL(EMEM(f), EBINOP(ADD, EMEM(a), ECSTI(1)) : ?, EMEM(b)) : ?) : ?
